README.md

```diff
@@ -152,392 +152,428 @@ For information about the governance of the Node.js project, see
 ### TSC (Technical Steering Committee)
 
 * [addaleax](https://github.com/addaleax) -
-**Anna Henningsen** &lt;anna@addaleax.net&gt; (she/her)
+**Anna Henningsen** [anna@addaleax.net](mailto:anna@addaleax.net) (she/her)
```

Comment:
If the mailto format isn't prefered here, the rule could be disabled at the file level

Reply:
I think node-core-utils might parse these lines of the README file for some operations. If I'm not wrong about that, then that would need to be tested.

Reply:
any idea how to test this or who to ping? I'm also good with just disabling at the file or block level

Reply:
@nodejs/automation-collaborators

Reply:
You could attempt to use `node-core-utils` to land a PR?

The parsing is done in https://github.com/nodejs/node-core-utils/blob/master/lib/collaborators.js.

Reply:
To avoid conflicts with the tooling, I've disabled the rule at the top of the README and reverted these changes.
I didn't actually test to see if there was an issue with the tool